vector/src/main/java/im/vector/app/features/location/LocationSharingFragment.kt

```diff
@@ -194,6 +195,25 @@ class LocationSharingFragment @Inject constructor(
         }
     }
 
+    private val liveLocationLabsFlagPromotionListener = object : VectorBaseBottomSheetDialogFragment.ResultListener {
+        override fun onBottomSheetResult(resultCode: Int, data: Any?) {
+            // Check if the user wants to enable the labs flag
+            if (resultCode == VectorBaseBottomSheetDialogFragment.ResultListener.RESULT_OK && (data as? Boolean) == true) {
```

Comment:
this comment above could be replaced by extracting this condition to its own function

```kotlin
private fun hasUserOptedInToLiveLocationLabs(code, data) {
  return resultCode == VectorBaseBottomSheetDialogFragment.ResultListener.RESULT_OK && (data as? Boolean) == true 
}
```

Reply:
Right, done.